pkg/subctl/cmd/verify.go

```diff
@@ -89,7 +93,7 @@ func configureTestingFramework(args []string) {
 	// the cluster IDs that will be registered in the Cluster CRDs by submariner
 	framework.TestContext.ClusterIDs = []string{"ClusterA", "ClusterB"}
 
-	config.GinkgoConfig.FocusString = "dataplane"
+	config.GinkgoConfig.FocusString = "dataplane*"
```

Comment:
We need to modify the FocusString based on connectivity / discovery

Reply:
> We need to modify the FocusString based on connectivity / discovery

This change doesn't work, I tried already. I am modifying it for discovery.